main.js

```diff
@@ -0,0 +1,50 @@
+
+const tweeter = Tweeter()
+const renderer = Renderer()
+
+renderer.renderPosts(tweeter.getPosts())
+
+
+
+function post() {
+    if (($('#input').val()).length > 0) {
+        tweeter.addPost($('#input').val())
+        $('#input').val('')
+        renderer.renderPosts(tweeter.getPosts())
+    }
+}
+
+
+
+$('#posts').on ('click', '.delete', function() {
+    tweeter.removePost($(this).closest($('.post')).find('.post-text').attr('id'))
+    renderer.renderPosts(tweeter.getPosts())
+}
+
+)
+
+
+
+$('#posts').on ('click', 'button', function() {
+    let commentText = $(this).closest($('.post')).find('input').val()
+    let postId = $(this).closest($('.post')).find('.post-text').attr('id')
+    tweeter.addComment(commentText, postId)
+    renderer.renderPosts(tweeter.getPosts())
+}
+
+)
+
+
+
+$('#posts').on ('click', '.delete-comment', function() {
+    tweeter.removeComment($(this).closest($('.post')).find('h3').attr('id'), $(this).closest($('div')).attr('id'))
```

Comment:
its a little bit too long, don't be afraid to use variables to hold some content for you)